pkg/cmd/stack/deploy.go

```diff
@@ -153,6 +154,30 @@ func deployDeployment(ctx context.Context, svcName string, s *model.Stack, c *ku
 }
 
 func deployStatefulSet(ctx context.Context, svcName string, s *model.Stack, c *kubernetes.Clientset) error {
+
+	pvcList := translatePersistentVolumeClaims(svcName, s)
+	for _, pvc := range pvcList {
+		old, err := c.CoreV1().PersistentVolumeClaims(s.Namespace).Get(ctx, pvc.Name, metav1.GetOptions{})
+		if err != nil && !errors.IsNotFound(err) {
+			return fmt.Errorf("error getting persistent volume claim of service '%s': %s", svcName, err.Error())
```

Comment:
In our error messages to the user, let's say just "volume" instead of "persistent volume claim"